File: activityAndlistviewOrnegi/app/src/main/java/com/example/activityandlistviewornegi/Main2Activity.java
package com.example.activityandlistviewornegi;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

public class Main2Activity extends AppCompatActivity {
    TextView text1;
    TextView text2;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main2);
        text1=(TextView) findViewById(R.id.text1);
        text2=(TextView) findViewById(R.id.text2);
        al();

    }
    public void al(){
        Bundle bundle=getIntent().getExtras();
        String isim1=bundle.getString("isim");
        String soyisim2=bundle.getString("soyisim");
        text1.setText(isim1);
        text2.setText(soyisim2);
    }
}

File: KotlinDers1StoringData/app/src/main/java/com/example/kotlinders1storingdata/Main2Activity.kt
package com.example.kotlinders1storingdata

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Toast
import androidx.appcompat.app.AlertDialog
import kotlinx.android.synthetic.main.activity_main2.*

class Main2Activity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main2)
        //textView2 de göster
        val intent=intent
        val ageGived=intent.getIntExtra("age",0)
        textView2.text=""+ageGived
        toastMessage()
    }
    fun toastMessage(){
        Toast.makeText(applicationContext,"merhabaa",Toast.LENGTH_LONG).show()
    }

    fun alert(view: View){
        //basıldığında. Alert açılacak
        val alert=AlertDialog.Builder(this)
        alert.setTitle("diggat")
        alert.setMessage("diggat et bebegim")
        alert.setPositiveButton("ederim"){dialog, which ->
            //ederime basılınca ne olacak
            Toast.makeText(applicationContext,"aferim",Toast.LENGTH_LONG).show()
        }
        //{} bu ve içindekiler lambda gösterimi
        alert.setNegativeButton("etmem"){dialog, which ->

        }
        alert.show()


    }
}
